modules/navier_stokes/include/base/NS.h

```diff
@@ -19,10 +19,11 @@ const std::string momentum_y = "rhov";
 const std::string momentum_z = "rhow";
 const std::string total_energy = "rhoE";
 
+const std::string velocity = "velocity";
 const std::string velocity_x = "vel_x";
 const std::string velocity_y = "vel_y";
 const std::string velocity_z = "vel_z";
-const std::string pressure = "pressure";
+const std::string pressure = "p";
```

Comment:
Seems like if we undid this change here, then my concern in https://github.com/idaholab/moose/pull/15305#discussion_r428940253 would go away and we wouldn't have to re-gold.

Reply:
Then INS action will be using pressure as the variable name and I will need to create more gold exodus files because of this.

Reply:
Using p here is actually having a more consistent style as other names in NS.h.